CHANGELOG.md

```diff
@@ -1,3 +1,527 @@
+## v0.4.0 - 2020-09-07
+
+We're happy to announce the release of Lokomotive v0.4.0 (Darjeeling Himalayan).
+
+This release packs new features, bug fixes, code optimizations, better user interface, latest
+versions of components, security hardening and much more.
+
+### Changes in v0.4.0
+
+#### Kubernetes Updates
+
+- Update Kubernetes version to
+  [`v1.18.8`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#v1188)
+  ([#861](https://github.com/kinvolk/lokomotive/pull/861)).
+
+#### Platform updates
+
+##### AKS
+
+- Update Kubernetes version to `1.17.9` ([#849](https://github.com/kinvolk/lokomotive/pull/849)).
+
+##### AWS
+
+- AWS: Add support for custom taints and labels
+  ([#832](https://github.com/kinvolk/lokomotive/pull/832)).
+
+#### New Components
+
+- Add component `experimental-istio-operator`
+  ([#686](https://github.com/kinvolk/lokomotive/pull/686)).
+- Add component `experimental-linkerd` ([#690](https://github.com/kinvolk/lokomotive/pull/690)).
+
+#### Component updates
+
+- Update etcd to `v3.4.13` ([#838](https://github.com/kinvolk/lokomotive/pull/838)).
+- Update Calico to `v3.15.2` ([#841](https://github.com/kinvolk/lokomotive/pull/841)).
+- Update Grafana to `7.1.4` and chart version `5.5.5`
+  ([#842](https://github.com/kinvolk/lokomotive/pull/842)).
+- Update Velero chart to `1.4.2` ([#830](https://github.com/kinvolk/lokomotive/pull/830)).
+- Update external-dns chart to `3.3.0` ([#845](https://github.com/kinvolk/lokomotive/pull/845)).
+- Update aws-ebs-csi-driver to `v0.6.0` ([#856](https://github.com/kinvolk/lokomotive/pull/856)).
+- Update cluster-autoscaler to `v2` version `1.0.2`
+  ([#859](https://github.com/kinvolk/lokomotive/pull/859)).
+- Update cert-manager to `v0.16.1` ([#847](https://github.com/kinvolk/lokomotive/pull/847)).
+- Update openebs to `v1.12.0` ([#781](https://github.com/kinvolk/lokomotive/pull/781)).
```

Comment:
We should consistently use either `component name` or Project name here I think.
```suggestion
- Update OpenEBS to `v1.12.0` ([#781](https://github.com/kinvolk/lokomotive/pull/781)).
```